app/src/main/res/values/styles.xml

```diff
@@ -1,19 +1,37 @@
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
 
-    <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.DayNight.DarkActionBar">
-        <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
+<!-- Base application theme. -->
+<style name="AppTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+    <!-- Customize your theme here. -->
+    <item name="colorPrimary">@color/colorPrimary</item>
+    <item name="colorPrimaryDark">@color/bgColor</item>
+    <item name="android:windowLightStatusBar" tools:targetApi="29">true</item>
+    <item name="colorAccent">@color/colorAccent</item>
+    <item name="bottomSheetDialogTheme">@style/AppBottomSheetDialogTheme</item>
+</style>
+    <style name="AppBottomSheetDialogTheme"
+            parent="Theme.Design.Light.BottomSheetDialog">
+        <item name="bottomSheetStyle">@style/AppModalStyle</item>
     </style>
+    <style name="AppModalStyle"
+            parent="Widget.Design.BottomSheet.Modal">
+        <item name="android:background">@drawable/rounded_dialog</item>
+    </style>
+<style name="FloatingLabelTextAppearance" parent="TextAppearance.AppCompat">
+    <item name="android:textColor">@android:color/white</item>
+</style>
 
-    <style name="FloatingLabelTextAppearance" parent="TextAppearance.AppCompat">
-        <item name="android:textColor">@android:color/white</item>
+<style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
```

Comment:
@coder2699 Why have you made changes in the splash theme?

Reply:
@rob729 There's no changes made in the splash screen theme. It has been highlighted because while writing other code some blank line might have been added unintentionally.